internal/config_checks_rp2040.h

```diff
@@ -85,14 +85,17 @@ MOZZI_CHECK_SUPPORTED(MOZZI_AUDIO_MODE, MOZZI_OUTPUT_EXTERNAL_TIMED, MOZZI_OUTPU
 
 #if MOZZI_IS(MOZZI_AUDIO_MODE, MOZZI_OUTPUT_PWM)
 #  if !defined(MOZZI_AUDIO_BITS)
-#    define MOZZI_AUDIO_BITS 11
+#    define MOZZI_AUDIO_BITS 16 // PWMAudio expects 16bits
 #  endif
 #  if !defined(MOZZI_AUDIO_PIN_1)
 #    define MOZZI_AUDIO_PIN_1 0
 #  endif
 #  if !defined(MOZZI_AUDIO_PIN_2)
 #    define MOZZI_AUDIO_PIN_2 1
 #  endif
+#  define BYPASS_MOZZI_OUTPUT_BUFFER true
+#  define MOZZI_RP2040_BUFFERS 8       // number of DMA buffers used
```

Comment:
It may be worth documenting: Is that 8 the result to experimentation, or would other values be expected to work, too?

Reply:
I used the same values that the library example (which are the same than for I2S). Other values might be expected to work, but I did not test thoughtfully.